### PR TITLE
Clean up EE skipped tests

### PR DESCRIPTION
**Pull request by amcasey**
Unskip the ones associated with closed bugs and make sure the others are
clearly labelled.

### CODE REVIEW

**Comment by amcasey** [comment]
FYI @cston @KevinH-MS 
@ManishJayaswal  should this go into stabilization?

File: src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/LocalsTests.vb [hunk 1]
```diff
@@ -1401,7 +1401,7 @@ End Class
             locals.Free()
         End Sub
 
-        <Fact(Skip:="1002672"), WorkItem(1002672)>
+        <Fact, WorkItem(1002672)>
```
Comment (KevinH-MS) [inline src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/LocalsTests.vb]: Consider adding "DevDiv"

File: src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/ExpansionTests.vb [hunk 2]
```diff
@@ -627,7 +627,7 @@ End Class
                 EvalResult("S", "42", "Integer", "A.S"))
         End Sub
 
-        <Fact(Skip:="1074435"), WorkItem(1074435)>
+        <Fact, WorkItem(1074435)>
```
Comment (KevinH-MS) [inline src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/ExpansionTests.vb]: Consider adding "DevDiv"

**Comment by KevinH-MS** [comment]
Thanks! :+1: 

As an aside, I wonder if we should have a WorkItemAttribute that just takes a string.  It seems redundant to have the id as an int and then have a url that contains the int.

**Comment by ManishJayaswal** [comment]
Thanks @amcasey . Yes this should go to stabilization.

File: src/ExpressionEvaluator/CSharp/Test/ResultProvider/DebuggerDisplayAttributeTests.cs [hunk 3]
```diff
@@ -332,7 +332,7 @@ class Wrapper
                 EvalResult("NoDisplayPointer", PointerToString(IntPtr.Zero), "NoDisplay*", "wrapper.display.NoDisplayPointer"));
         }
 
-        [Fact(Skip = "Issue #321")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/321")]
         public void PointerDereferenceExpansion_NonNull()
```
Comment (cston) [inline src/ExpressionEvaluator/CSharp/Test/ResultProvider/DebuggerDisplayAttributeTests.cs]: [WorkItem(321)]

**Comment by cston** [comment]
LGTM

File: src/ExpressionEvaluator/CSharp/Test/ResultProvider/DebuggerDisplayAttributeTests.cs (hunk 3 above)
Comment (amcasey) [inline src/ExpressionEvaluator/CSharp/Test/ResultProvider/DebuggerDisplayAttributeTests.cs]: I included the URL as well.

File: src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/LocalsTests.vb (hunk 1 above)
Comment (amcasey) [inline src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/LocalsTests.vb]: Done.

File: src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/ExpansionTests.vb (hunk 2 above)
Comment (amcasey) [inline src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/ExpansionTests.vb]: Done.

**Comment by amcasey** [comment]
I'm going to close this PR since I can't update it to merge into stabilization.

**Pull request closed without merging**